src/fonts/textfonts.ts

```diff
@@ -0,0 +1,56 @@
+// [VexFlow](https://vexflow.com) - Copyright (c) Mohit Muthanna 2010.
+// MIT License
+
+import { Font } from '../font';
+import { TextFormatter } from '../textformatter';
+import { PetalumaScriptFont } from './petalumascript_glyphs';
+import { RobotoSlabFont } from './robotoslab_glyphs';
+
+export function loadTextFonts() {
+  // Roboto Slab
+  {
+    const fontData = RobotoSlabFont;
+    const { fontFamily, resolution, glyphs } = fontData;
+    Font.load(fontFamily, fontData);
+    // H isn't actually the tallest or the widest.
+    // Interestingly, the lowercase b is the tallest.
+    const maxSizeGlyph = 'H'; // Two lines of lyrics touch each other because we don't allow enough vertical space.
+    // const maxSizeGlyph = 'b'; // RONYEH: introduces a visual diff, but the spacing is better.
+    TextFormatter.registerInfo({
+      family: fontFamily,
+      resolution,
+      glyphs,
+      maxSizeGlyph,
+      monospaced: false,
+      bold: false,
+      italic: false,
+      superscriptOffset: 0.66,
+      subscriptOffset: 0.66,
+      serifs: true,
+      description: 'Text font to pair with the Bravura / Gonville music fonts.',
+    });
+  }
```

Comment:
@AaronDavidNewman will probably recognize this code. It was extracted from the previous TextFont class, and now parallels the loadMusicFonts() function that we have.

You'll need to look for TextFormatter (as we discussed a while back), as the TextFont class is no longer around.